### ISSUE TITLE
Machinaris shows server error 500 in plotting mode

### ISSUE THREAD

**Issue by raskitoma** [guydavis/machinaris#126]
**Describe the bug**
Setting up Machinaris as a plotter results on web ui as Server Error 500

Using docker via docker-compose:

1. Create file `machinaris.yml`:

```
version: '3'
services:
  machinaris:
    container_name: machinaris
    image: ghcr.io/guydavis/machinaris
    restart: always
    tty: true
    ports:
      - "8444:8444"
      - "8926:8926"
    volumes:
      - "/home/user/machinaris:/root/.chia:rw"
      - "/home/user/chia/temp:/plotting:rw"
      - "/home/user/chia/temp2:/plotting2:rw"
      - "/home/user/chia/plots:/plots:rw"
    environment:
      - TZ=America/New_York
      - mode=plotter
      - pool_pk=mysecretkey
      - farmer_pk=anothersecretkey
```

2. Setup and start container with `docker-compose -f machinaris.yml up -d`

3. Go to web browser, type `http://machinarishost:8926`
4. Error displayed on browser window body:  **HTTP Error 500 - Internal Server Error**

**Expected behavior**

Instead of HTTP Error 500, it should show the Summary

**System setup:**
 - OS: Ubuntu Server 20.04
 - Docker version number 20.10.2
 - Machinaris branch: main
 - Machinaris version number 0.4.0

**Config**

Config default, clean state.

</details>

**Additional context & screenshots**

I notice in webui.log, using `docker exec -it machinaris tail -f /root/.chia/machinaris/logs/webui.log` from a fresh Machinaris installation that you receive the following error:

```
 File "/chia-blockchain/venv/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
    raise exception
  File "/chia-blockchain/venv/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 605, in __connect
    connection = pool._invoke_creator(self)
  File "/chia-blockchain/venv/lib/python3.9/site-packages/sqlalchemy/engine/create.py", line 578, in connect
    return dialect.connect(*cargs, **cparams)
  File "/chia-blockchain/venv/lib/python3.9/site-packages/sqlalchemy/engine/default.py", line 584, in connect
    return self.dbapi.connect(*cargs, **cparams)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
(Background on this error at: http://sqlalche.me/e/14/e3q8)
```

As you can see above, the databases cannot be found/opened.

I found a closed [issue](https://github.com/guydavis/machinaris/issues?q=is%3Aissue+docker+version+test+is%3Aclosed) that point me at the right direction about the error.

 After running the following commands:

`docker exec -it machinaris /machinaris/scripts/setup_databases.sh`

`docker restart machinaris`

Machinaris works as expected.

**Comment by guydavis** [comment]
Yes, thanks for the report.  I've enabled the setup_databases.sh script again for standalone plotter-mode.  Already available in the test release: `ghcr.io/guydavis/machinaris:test`  

However, I would recommend trying a Machinaris Controller to remotely manage the Machinaris Plotter system, rather than the old approach of a standalone plotter.  The new [Workers](https://github.com/guydavis/machinaris/wiki/Workers) functionality in v0.4.0 does not have this issue.

**Comment by guydavis** [comment]
Closing as I've fixed standalone plotter mode in [v0.5.0](https://github.com/guydavis/machinaris/discussions/159), however I still recommend [Workers](https://github.com/guydavis/machinaris/wiki/Workers) functionality, rather than standalone mode.
